qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -41,12 +41,23 @@ def remove_snapshots(self, dir_path, sdn):
             self.mount_a.run_shell(['sudo', 'rmdir', snapshot_path])
 
     def get_snap_dir_name(self):
-        from .fuse_mount import FuseMount
-        from .kernel_mount import KernelMount
+        km = None
+        fm = None
+
+        if 'Local' in str(type(self.ceph_cluster)):
+            from tasks.vstart_runner import LocalFuseMount
```

Comment:
You should not be importing vstart_runner from the core test code. that's against the whole point of dependency injection.